change/@internal-react-composites-81715f29-3fde-4133-99e8-70378241e624.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
cc @prprabhu-ms bug or intentional this only created a stable changefile for this changfile?

Reply:
(would expect to see both a stable and a beta changelog)

Reply:
Probably needs a merge from `main`.

@dmceachernmsft if you think this branch might be old, could you (1) merge from `main` (2) run `rush changelog` again; (3) delete this file because the last step will create duplicate change-files.